Fix mutation logic (#226)
giantswarm/app-admission-controller: CHANGELOG.md

## [Unreleased]

> ### Fixed
> 
> - Fix mutation logic for `giantswarm.io/cluster` label for org-namespaced App CRs.

Reviewer: ```suggestion
### Changed

- Adapt mutation logic for `giantswarm.io/cluster` label to support App CRs in the org namespace.
```

Nit but I'd say supporting the org namespace is new functionality rather than a fix.

Author: Makes sense, thanks!
